### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by dan-auth0**
### Changes

- Fix typo. Capitalize "auth0"

### References

- "Auth0" is the proper trade name: [https://auth0.com/](https://auth0.com/)

### Testing

Not applicable.

### Checklist

[ X ] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)

[ X ] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)

[ X ] All existing and new tests complete without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,15 +107,22 @@ app.get('/login',
...
 
 ## Complete example
 
-A complete example of using this library [here](http://github.com/auth0/passport-auth0).
+A complete example of using this library is shown [here](http://github.com/auth0/passport-auth0).
 
 ## Documentation
 
-For more information about [auth0](http://auth0.com) contact our [documentation page](http://docs.auth0.com/).
+For more information about [Auth0](http://auth0.com) contact our [documentation page](http://docs.auth0.com/).
```
Comment (joshcanhelp) [inline README.md]: > ... contact our documentation page

Funny way of putting that :) 

> http://docs.auth0.com/

Wrong URL 

I wonder if we shouldn't point them somewhere else? Docs does not have anything about Passport that I know of.

File: README.md [hunk 2]
```diff
@@ -107,15 +107,22 @@ app.get('/login',
...
+
+You can also see fully working demos using this library in our [Auth0 blog](https://auth0.com/blog/):
+
+* [Build and Authenticate a Node.js App with JSON Web Tokens](https://auth0.com/blog/building-and-authenticating-nodejs-apps/#nodejs-directory-structure)
+
```
Comment (joshcanhelp) [inline README.md]: Remove whitespace

File: README.md [hunk 3]
```diff
@@ -107,15 +107,22 @@ app.get('/login',
 
 ## API access
 
-If you want to get a list of connections or users from auth0, use the [auth0 module](https://github.com/auth0/node-auth0).
+If you want to get a list of connections or users from Auth0, use the [Auth0 module](https://github.com/auth0/node-auth0).
 
 ## Complete example
 
-A complete example of using this library [here](http://github.com/auth0/passport-auth0).
+A complete example of using this library is shown [here](http://github.com/auth0/passport-auth0).
```
Comment (joshcanhelp) [inline README.md]: Is this necessary? That's the page they might be on if they're in GitHub. Maybe a specific anchor link?

File: README.md [hunk 4]
```diff
@@ -107,15 +107,22 @@ app.get('/login',
 
 ## API access
 
-If you want to get a list of connections or users from auth0, use the [auth0 module](https://github.com/auth0/node-auth0).
+If you want to get a list of connections or users from Auth0, use the [Auth0 module](https://github.com/auth0/node-auth0).
```
Comment (joshcanhelp) [inline README.md]: > ... use the Node.js SDK

File: README.md [hunk 5]
```diff
@@ -76,7 +76,7 @@ app.get('/login',
 });
 ~~~
 
-This way when you go to ```/login``` you will get redirected to auth0, to a page where you can select the identity provider.
+This way when you go to ```/login```, you will get redirected to an Auth0 page where you can select the identity provider.
```
Comment (joshcanhelp) [inline README.md]: > ```/login```

Extra backticks

File: README.md (hunk 1 above)
Comment (dan-auth0) [reply README.md]: @joshcanhelp Yeah, that part was reading wrong to me. I think here, we could rather prompt them to sign up.

What about something like this:

## Auth0: Never Compromise on Identity

At [Auth0](https://auth0.com/), we make heavy use of full-stack JavaScript to help our customers [manage user identities which includes password resets along with creating, provisioning, blocking, and deleting users](https://auth0.com/user-management). Therefore, it must come as no surprise that using our identity management platform on JavaScript web apps is a piece of cake.

[Auth0 offers a **free tier**](https://auth0.com/pricing) to get you started with modern authentication. Check it out, or <a href="https://auth0.com/signup" data-amp-replace="CLIENT_ID" data-amp-addparams="anonId=CLIENT_ID(cid-scope-cookie-fallback-name)">sign up for a free Auth0 account here</a>!

Auth0 Login Page

You may also check out our [Quick Start tutorials](https://auth0.com/docs/quickstarts) to learn how to implement authentication using different languages and frameworks in your apps.

File: README.md (hunk 3 above)
Comment (dan-auth0) [reply README.md]: True!

File: README.md (hunk 1 above)
Comment (joshcanhelp) [reply README.md]: Here's the standard blurb we use on our SDKs:

https://github.com/auth0/open-source-template/blob/master/README-sample.md#what-is-auth0

That's content-approved so should be fine to use anywhere!

**Pull request merged**